package.json

```diff
@@ -54,13 +54,13 @@
   ],
   "dependencies": {
     "@typescript-eslint/parser": "^5.50.0",
-    "eslint-config-standard": "17.0.0"
+    "eslint-config-standard": "17.1.0"
   },
   "peerDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.50.0",
     "eslint": "^8.0.1",
     "eslint-plugin-import": "^2.25.2",
-    "eslint-plugin-n": "^15.0.0",
+    "eslint-plugin-n": "^15.0.0 || ^16.0.0 ",
```

Comment:
Note this extra space comes from `eslint-config-standard`, we can remove it but the test will fail in our end.

https://github.com/standard/eslint-config-standard/blob/7d01bbc373bf3bb7374b698323032077ad05f6b1/package.json#L57